src/autopas/AutoPasImpl.h

```diff
@@ -203,4 +203,9 @@ std::shared_ptr<const autopas::ParticleContainerInterface<Particle>> AutoPas<Par
   return _autoTuner->getContainer();
 }
 
+template <class Particle>
+bool AutoPas<Particle>::getsearchSpaceIsTrivial() {
+  return _autoTuner->searchSpaceIsTrivial();
+}
+
```

Comment:
Have you checked whether it is possible to have this function in `AutoPasDecl.h`? I don't think it is but just for my peace of mind :)

Reply:
I already checked this and sadly couldn't get it to work. I think it has something to do with referencing AutoTuner.

Reply:
👍